README.md

```diff
@@ -9,8 +9,8 @@ However, the ObjectBox core supports many more features, e.g. queries, indexing,
 To bring all these features to Dart, we're asking the community to help out. PRs are more than welcome!
 The ObjectBox team will try its best to guide you and answer questions. 
 
-### Open Development Process
-
+Contributing
```

Comment:
Add a special GitHub CONTRIBUTING file (shown when creating issues/pull requests)?
https://help.github.com/en/articles/setting-guidelines-for-repository-contributors

Reply:
I'll create an issue